automagicallyexecute/src/executeAutomagically.ts

```diff
@@ -41,6 +42,7 @@ export const executeAutomagically = async ({
   const token = getInputRequired('token')
   const testTargetId = getInputRequired('testTargetId')
   const environmentName = getInput('environmentName')
+  const tags = getDelimitedInput('tags', '\n')
```

Comment:
uno unit testo

Reply:
i will buy you a beer if this unit test provides any value ;)